lib/array.gb

```diff
@@ -4,4 +4,8 @@ class Array
       i == x
     end
   end
+
+  def to_enum
+		ArrayEnumerator.new(self)
```

Comment:
@saveriomiroddi  Can you make this indentation right? After that I think we'll be ok to merge this